File: src/i18n/en_in.go
package i18n

// enINOverrides is empty. Indian English follows British spelling, which
// inheriting en-GB supplies. Where Indian English is genuinely distinctive —
// lakh and crore for large numbers, its own register of official phrasing —
// this catalog has no numbers spelled as words and no official register to
// depart from.
var enINOverrides = map[string]string{}

// enINMessages is en-GB unchanged, reviewed for Indian usage.
var enINMessages = derive(enGBMessages, enINOverrides)

File: ui/src/i18n/en-IN.js
import derive from './derive.js'
import enGB from './en-GB.js'

/**
 * Indian English follows British spelling, supplied by inheriting en-GB. Where
 * Indian English is genuinely distinctive — lakh and crore for large numbers,
 * its own register of official phrasing — this catalog spells no number as a
 * word and has no official register to depart from.
 *
 * Empty is the finding, not a gap.
 */
export const enINOverrides = {}

/** English (India) — en-GB unchanged, reviewed for Indian usage. */
const enIN = derive(enGB, enINOverrides)

export default enIN
